### PR TITLE
WIP: Add JuliaFormatter and remove acronyms

### PR DESCRIPTION
**Pull request by odow**
Should be rebased after #278

### CODE REVIEW

File: src/MOI_wrapper.jl [hunk 1]
```diff
@@ -1,29 +1,16 @@
-using MathOptInterface
-const MOI = MathOptInterface
-const MOIU = MOI.Utilities
```
Comment (matbesancon) [inline src/MOI_wrapper.jl:3]: MOIU is standard no?

Comment (odow) [reply src/MOI_wrapper.jl:3]: I really don't like it. `MOI.Utilities` is much clearer, and you very rarely need to use `MOIU` anyway.

**Comment by odow** [comment]
One downside to this PR is that it will screw up all the open PRs. So I'll rebase, but don't feel pressured to merge

**Comment by matbesancon** [comment]
that's fine, all of them are inactive or just trials that will not be merged for now

**Pull request merged**